indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -23,22 +23,26 @@
 {% macro render_data(registration, field, from_management=false) %}
     {% set data = registration.data_by_field -%}
     {% set friendly_data = data[field.id].friendly_data if field.id in data else '' -%}
-    {% if field.input_type == 'accommodation' and friendly_data %}
-        <div>
-            {% trans arrival_date=friendly_data['arrival_date'] | format_date -%}
-                <strong>Arrival:</strong> {{ arrival_date }}
-            {%- endtrans %}
-        </div>
-        <div>
-            {% trans departure_date=friendly_data['departure_date'] | format_date -%}
-                <strong>Departure:</strong> {{ departure_date }}
-            {%- endtrans %}
-        </div>
-        <div>
-            {% trans accommodation=friendly_data['choice'] -%}
-                <strong>Accommodation:</strong> {{ accommodation }}
-            {%- endtrans %}
-        </div>
+    {% if field.input_type == 'accommodation' %}
+        {% if friendly_data %}
+            <div>
+                {% trans arrival_date=friendly_data['arrival_date'] | format_date -%}
```

Comment:
I know it was like this before, but it'd be a good opportunity to change this to `friendly_data.arrival_date` (same for the other ones below)